File: src/main/java/com/github/alexandrecarlton/idea/settings/layout/editor/codestyle/java/wrapping_and_braces/Force.java
package com.github.alexandrecarlton.idea.settings.layout.editor.codestyle.java.wrapping_and_braces;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum Force {
  @JsonProperty("Do not force")
  DO_NOT_FORCE,

  @JsonProperty("When multiline")
  WHEN_MULTILINE,

  @JsonProperty("Always")
  ALWAYS
}

File: layout/src/main/kotlin/com/github/alexandrecarlton/idea/settings/dialog/editor/codestyle/java/wrapping_and_braces/Force.kt
package com.github.alexandrecarlton.idea.settings.dialog.editor.codestyle.java.wrapping_and_braces

import com.fasterxml.jackson.annotation.JsonProperty

enum class Force {
    @JsonProperty("Do not force")
    DO_NOT_FORCE,

    @JsonProperty("When multiline")
    WHEN_MULTILINE,

    @JsonProperty("Always")
    ALWAYS
}
